### DISCUSSION TITLE
Integration with Amari Callbox

### DISCUSSION THREAD

**Discussion by Mariakhanmm92** [aligungr/UERANSIM#410] [Q&A] [answered]
Hello @aligungr and other contributors,

I want to integrate the UERANSIM's 5G SA UE with Amarisoft's Callbox (5GC+gNB). Is it possible? Is there anyone who has tried it already? that will be quite helpful for me.

Thanks in advance!

**Comment by aligungr** [comment, marked as answer]
Hello Maria

Our UE cannot be used with an external gNodeB as we have our custom protocols in radio layer. (Between UE and gNodeB)

But you can use external core networks with our UE or gNodeB.

**Comment by Mariakhanmm92** [comment]
Thanks for the prompt response. I will look into it.
